_pages/docs/how-does-it-work.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: inner
+title: How Does it Work?
+lead_text: ''
+permalink: /docs/how-does-it-work/
+---
+
+# How Does it Work?
+
+Smart Mastering consists of configuration-driven matching and merging. While
+the [How To Use][how-to-use] page will show you how to access this
+functionality, this page describes the matching and merging processes that
+make up Smart Mastering.
+
+The Smart Mastering process focuses on one document at a time, identifying
+any other documents that appear to represent the same entity (Matching) and
+then combining the values in those documents to create a new document (Merging).
+
+## Collections
+
+Smart Mastering Core uses collections to separate types of content. Applications
+should use [`constants.xqy`][constants] for the names of the collections. The
+most important collection is `$CONTENT-COLL`, which contains the current set of
+entities that should be used by an application. When a set of documents get
+merged, they are moved out of that collection and into the `$ARCHIVED-COLL`
+collection. The generated merged document will be in the `$CONTENT-COLL` and
+`$MERGED-COLL` collections. 
+
+## Matching
+
+The matching process begins with a document, which we'll call the "original"
+document. This document may be selected because it's just been inserted into the
```

Comment:
I'd like to see this broken out into numbered steps that make it a bit easier to digest. Maybe even a picture.

That way I can get the high level quickly. If I need to know about cts:queries then I can read the details under each step.